act-rules-format.bs

```diff
@@ -37,7 +37,7 @@ In accessibility, there are often different technical solutions to make the same
 
 - <dfn>Atomic rules</dfn> describe how to test a specific type of solution. It contains a precise definition of what elements, nodes or other parts of a [=test subject=] are to be tested, and when those elements are considered to fail the rule. These rules are to be kept small and *atomic*. Meaning that atomic rules test a single "failure condition", and do so without using the [=outcome=] of other rules.
 
-- <dfn>Composite rules</dfn> describe how the [=outcome=] of [=atomic rules=] is used to decide if an accessibility requirement is (non-)conformant. If this involves multiple "failure conditions", each of these is provided in a separate atomic rule. The logic for determining (non-)conformance to the accessibility requirement according to outcome of the individual the atomic rules is specified in the composite rule.
+- <dfn>Composite rules</dfn> describe how the [=outcome=] of multiple [=atomic rules=] can be combined into a single outcome. A composite rule can have multiple "satisfying conditions", each of these provided in a separate atomic rule. The logic in the composite rule describes how any one of these satisfying conditions, or some combination of them is used to determine a single outcome.
```

Comment:
- I really like the idea of saying "satisfying conditions" because it is not limited to "failure" or "success" but could be any. I think we need to reflect that phrasing in the atomic rules definition too. That is, "satisfying condition" instead of "failure condition".
- The last sentence seems to repeat the first two. Actually it contradicts them somehow -- is it "multiple" or "any one or more"? I'm not sure what the last sentence adds that is not already in the first two, and think it should be removed.
- Following the previous point, I think the word "multiple" is wrong in the first two sentences, regardless if you keep the third sentence or drop it. It should read "any one or more" instead of "multiple".

Reply:
Suggest the last sentence reads, "The logic in the composite rule describes one or more of these satisfying conditions is used to determine a single outcome."